blueprints/app/files/testem.js

```diff
@@ -11,11 +11,14 @@ module.exports = {
     Chrome: {
       mode: 'ci',
       args: [
+        // --no-sandbox is needed when running Chrome inside a container
+        process.env.TRAVIS ? '--no-sandbox' : null,
```

Comment:
Should we use `process.env.CI` instead? As I understand it a number of CI providers have this same basic issues (e.g. GitLab CI), and the CI environment flag is pretty universal...

Reply:
I'm not sure... GitLab CI can and often does use Docker containers, but you can also run it on real hosts, AppVeyor is likely VM-based, so there are also some CI systems that do not run in containers. The condition is easy enough to adjust if needed 🤔